frontend/src/metabase/nav/components/PathBreadcrumbs/PathBreadcrumbs.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import Icon from "metabase/components/Icon";
+import Collection from "metabase/entities/collections";
+import CollectionBadge from "metabase/questions/components/CollectionBadge";
+import { Collection as CollectionType } from "metabase-types/api/collection";
+import { EntitiesState } from "metabase-types/store/entities";
+
+import {
+  PathSeparator,
+  PathContainer,
+  ExpandButton,
+} from "./PathBreadcrumbs.styled";
+
+interface Props {
+  collection: CollectionType;
+  collectionId: string;
```

Comment:
You can also use `CollectionId` type from `metabase-types/api/collection`. Collection ID can be an integer or `null` / `"root"` string for "Our analytics"